packages/flutter_test/lib/src/matchers.dart

```diff
@@ -366,9 +367,9 @@ Matcher coversSameAreaAs(Path expectedPath, { @required Rect areaToCompare, int
 ///    may swap out the backend for this matcher.
 AsyncMatcher matchesGoldenFile(dynamic key, {int version}) {
   if (key is Uri) {
-    return _MatchesGoldenFile(key, version);
+    return MatchesGoldenFile(key, version);
```

Comment:
Is there potential for confusion between `matchesGoldenFile` and `MatchesGoldenFile` now that this will be publicly exposed?

Reply:
This is still not "publicly exposed" bacause it's not exported from `flutter_test.dart`

We moved platform-specific implementations into a separate file and use conditional imports to conditionally import the files, which is why we had to make it "public".

cc @jonahwilliams , are there other best practices for this scenario?

Reply:
Generally the way I've been doing it is:

```
_library_io.dart
_library_web.dart
library.dart (with conditional import/export of the above)
```

This makes it extra obvious that one should not import the _io or _web libraries too

Reply:
sgtm!